### PR TITLE
Provide both libdaq 2.2.2 and 3.0.0-beta1 to satisfy Snort 2 and 3.

### PR DESCRIPTION
**Pull request by flyn-org**
Maintainer: me
Compile tested: mips_24kc master

Description:
Earlier bump of libdaq left Snort 2 unable to compile. This provides both libdaq and libdaq3 packages so that both Snort 3 and 3 compile.

### CODE REVIEW

File: libs/libdaq3/Makefile [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+PKG_NAME:=libdaq3
+PKG_VERSION:=3.0.0-beta1
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://www.snort.org/downloads/snortplus/
+PKG_SOURCE:=daq-$(PKG_VERSION).tar.gz
+PKG_HASH:=ef74aa1c30a6ee93eacbe7967d1c85d7df3214cf3783d4eabbb6b64305fd273e
+PKG_BUILD_DIR:=$(BUILD_DIR)/daq-$(PKG_VERSION)
+
+PKG_LICENSE:=GPL-2.0
```
Comment (neheb) [inline libs/libdaq3/Makefile:19]: should be -only or -or-later .

File: libs/libdaq3/Makefile [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+PKG_MAINTAINER:=W. Michael Petullo <mike@flyn.org>
+
+PKG_INSTALL:=1
```
Comment (neheb) [inline libs/libdaq3/Makefile:22]: Should add PKG_BUILD_PARALLEL:=1

File: libs/libdaq3/Makefile [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
```
Comment (neheb) [inline libs/libdaq3/Makefile:25]: Is NLS actually used here?

File: libs/libdaq3/Makefile (hunk 1 above)
Comment (flyn-org) [reply libs/libdaq3/Makefile]: Fixed.

File: libs/libdaq3/Makefile (hunk 2 above)
Comment (flyn-org) [reply libs/libdaq3/Makefile:22]: Fixed.

File: libs/libdaq3/Makefile (hunk 3 above)
Comment (flyn-org) [reply libs/libdaq3/Makefile]: Fixed.

**Pull request merged**